### ISSUE TITLE
JS: Uncaught type error

### ISSUE THREAD

**Issue by baumanno** [plaperdr/fp-central#1]
Hey,

running the JS tests from https://fpcentral.irisa.fr/fp in Chromium results in this error, preventing execution from finishing, ultimately resulting in being unable to submit any stats.

Chromium: `Version 51.0.2704.79 Ubuntu 16.04 (64-bit)`

```
Uncaught TypeError: Cannot read property 'constructor' of undefined
    api.addTable @ clientAPI.js:101
    api.addTable @ clientAPI.js:103
    api.run @ clientAPI.js:128
    onclick @ fp:58
```

The last computed value is `screen.availWidth`, the first missing is `screen.left`.

**Comment by plaperdr** [comment]
Hey,
Thanks a lot for the bug report! I have noticed the exact same error on my end. I have designed the site to be first and foremost for the Tor browser and Firefox and it seems that I was so focused on them that I didn't even try with another browser. I don't have an ETA for a possible fix but I'll let you know as soon as I have one.

**Comment by plaperdr** [comment]
Hey,
I've normally fixed the bug in commit 90b9ff1279d433eb57c8492aace425822c4d7fc8.
It appeared that Chrome has no "screen.left" or "screen.right" properties. When the JS script was populating the HTML table with the result of one of these two properties, it crashed on a null value.
If you can check that it is okay on your end, I'll be able to close the issue.

**Comment by baumanno** [comment]
Hey,
cool stuff, I can confirm it works in Chrome 52.0.2743.82, albeit without values for screen.left/right. But according to your feedback, that is to be expected. 
Happy hacking!
